osgar/bus.py

```diff
@@ -56,6 +62,9 @@ def __init__(self, logger, name):
         self.finished = threading.Event()
         self.no_output = set()
         self.compressed_output = set()
+        self.time = None
```

Comment:
maybe use `self._time` to reduce confusion (as discussed)

Reply:
rename done